packages/esm-patient-attachments-app/src/ui-components/modal/modal.component.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+export const Modal: React.FC = ({ children }) => {
+  const [container, setConatiner] = React.useState<HTMLElement>(null);
+
+  React.useEffect(() => {
+    const div = document.body.appendChild(document.createElement('div'));
```

Comment:
This is a really weird modal.

Why not stay in the React tree and have a CSS class with `position: fixed`?

Besides, prefer to use a CSS class (module) instead of setting the style directly.